### ISSUE TITLE
Hepatic lobular inflammation

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6244]
**Preferred term label:**
Hepatic lobular inflammation

**Synonyms**

**Definition (free text, please give PubMed ID)**
Infiltration of inflammatory cells in lobules of the liver
A focus of lobular inflammation can be defined as two or more inflammatory cells (neutrophils, lymphocytes and other mononuclear cells, eosinophils and microgranulomas) within the lobule present within the sinusoids or surrounding injured hepatocytes (ballooned or apoptotic hepatocytes).

PMID: 31583739

comment: Each liver lobule is hexagonal in cross section and is centered on the central vein (teh central veins are branches of the hepatic vein). Within lobules, hepatocytes are arranged into hepatic cords separated by adjacent sinusoids.  

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
